### ISSUE TITLE
selector change

### ISSUE THREAD

**Issue by Qubus0** [joaoiacillo/chatgpt-messages-downloader#1]
hello, the selector for the message container has changed from
`#__next > div > div.flex.flex-1.flex-col.md\\:pl-52.h-full > main > div.flex-1.overflow-hidden > div > div > div`
to
`#__next > div > div.flex.h-full.flex-1.flex-col.md\\:pl-60 > main > div.flex-1.overflow-hidden > div > div > div`

**Comment by Qubus0** [comment]
well never mind, it just changed again to
`#__next > div > div.flex.h-full.flex-1.flex-col.md\\:pl-\\[260px\\] > main > div.flex-1.overflow-hidden > div > div > div`

for anyone getting this error:
`Uncaught TypeError: Cannot read properties of null (reading 'children')`
this selector is the issue. It is supposed to be the flexbox container in which all the messages are

**Comment by joaoiacillo** [comment]
Yep, it trully raises a TypeError. I'll be fixing it later. Thank you!
